x-pack/plugins/observability_solution/infra/public/components/asset_details/charts/chart.tsx

```diff
@@ -27,17 +28,24 @@ export type ChartProps = LensConfig &
 export const Chart = ({ id, queryField, overrides, dateRange, assetId, ...props }: ChartProps) => {
   const { setDateRange } = useDatePickerContext();
   const { searchSessionId } = useLoadingStateContext();
-  const { dataView } = useDataView({ index: (props.dataset as LensDataviewDataset)?.index });
+  const {
+    services: { dataViews },
+  } = useKibanaContextForPlugin();
+
+  const { value: filters = [] } = useAsync(async () => {
+    const resolvedDataView = await resolveDataView({
```

Comment:
it's because we need to retrieve the logs data view too for the logs rate chart. 

This chart component is data view agnostic and it depends on the index pattern information that the chart objects hold. This is only needed to build filters in a way that they look nicer when the user opens the chart in Lens. If it wasn't for that, this whole section could've been removed.